scripts/quests/otherAreas/The_Sand_Charm.lua

```diff
@@ -0,0 +1,143 @@
+-----------------------------------
+-- The Sand Charm
+-----------------------------------
+-- Log ID: 4, Quest ID: 8
+-- !addquest 4 8
+-- Blandine  : !pos 23 -7 41 249
+-- Zexu      : !pos 31.511 -9.001 23.496 249
+-- Celestina : !pos -37.624 -16.050 75.681 249
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.THE_SAND_CHARM)
+
+quest.reward =
+{
+    exp      = 2000,
+    gil      = 2000,
```

Comment:
May want to add this in here as well - 

https://github.com/LandSandBoat/server/blob/base/settings/default/main.lua#L118

Reply:
https://github.com/LandSandBoat/server/blob/base/settings/default/main.lua#L121 (EXP_RATE)

Reply:
Are you saying to multiply by the servers gil and exp rates?  I was under the impression that was handled by the framework.
https://github.com/LandSandBoat/server/blob/476a39e64c14aea3fcd317b00d7f8c0c981dc3d3/scripts/globals/npc_util.lua#L644

Reply:
You are right - 

```
    local currencyTypes =
    {
        ['gil']   = { 'GIL_OBTAINED', xi.settings.main.GIL_RATE },
        ['bayld'] = { 'BAYLD_OBTAINED', xi.settings.main.BAYLD_RATE }
    }
```

I was looking at the non framework ones.. proceed sir.